### ISSUE TITLE
How to make zsh path expansion case insensitive?

### ISSUE THREAD

**Issue by Praful** [junegunn/fzf#3325]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x] zsh
    - [ ] fish

## Problem / Steps to reproduce
I noticed that tab completions were not case sensitive in zsh and looked at the bindings. `^I` is bound to `fzf-completion`. I assume that means FZF is handling completions on the command line?

If so, is it possible to complete case insensitively?

If I type
```
ls p<TAB>
```
No expansion occurs. 

If I type (with a capital P):
```
ls P<TAB>
```
The line is expanded to
```
ls Pictures
```
My variables are:
```
export FZF_DEFAULT_COMMAND='rg --hidden -l ""'
export FZF_DEFAULT_OPTS="--ansi"
```
I've tried unsetting them and adding `-i` to `FZF_DEFAULT_COMMAND`.

How do I make the completion case insensitive?

**Comment by junegunn** [comment]
> that means FZF is handling completions on the command line?

Not really. The completion function of fzf first checks if there is a completion prefix (`**`) before the cursor, and if it's not found, it executes the usual `expand-or-complete` of zsh.

https://github.com/junegunn/fzf/blob/3e9efd1401404da8afdcf9757ac9996f5c48290f/shell/completion.zsh#L306-L309

So you need to find out how to configure case sensitivity in zsh.
